x-pack/metricbeat/module/stan/channels/_meta/fields.yml

```diff
@@ -0,0 +1,31 @@
+- name: channels
+  type: group
+  description: >
+    Contains stan / nats streaming/serverz endpoint metrics
+  release: beta
+  fields:
+    - name: name
+      type: keyword
+      description: >
+        The name of the STAN streaming channel
+    - name: messages
+      type: long
+      description: >
+        The number of STAN streaming messages
+    - name: bytes
+      type: long
+      description: >
+        The number of STAN bytes in the channel
+# If first_seq > min([seq in subscriptions]) data loss has possibly occurred
```

Comment:
Don't know if I've ever seen comments in fields files before? Might be worth seeing if you can cram this into the `description` field?

Reply:
Would a better place for an explanation of the metric field be better in the Go code rather than fields? Wasn't aware that field YAML files generally don't include comments

Reply:
For now I moved it in the description. Feel free to change it!